doc/building_blocks/data_preparation.rst

```diff
@@ -77,6 +77,13 @@ default algorithm performs poorly. This is why it is very important to
 Computing the mask
 ...................
 
+.. note::
+   
+    The full example described in this section can be found here:
+    :doc:`plot_mask_computation.py <../auto_examples/manipulating_visualizing/plot_mask_computation>`.
+    This one can be relevant too:
```

Comment:
Note aligns on the same line that image. I figured out that in other parts of doc such notes are in the beginning of section. So I moved it just after the header for consistency and nice html display.